homeassistant/components/switchbot/manifest.json

```diff
@@ -39,5 +39,5 @@
   "documentation": "https://www.home-assistant.io/integrations/switchbot",
   "iot_class": "local_push",
   "loggers": ["switchbot"],
-  "requirements": ["PySwitchbot==0.51.0"]
+  "requirements": ["PySwitchbot==0.53.0"]
```

Comment:
Please bump the dependency in a preliminary PR

Reply:
But can you please open a prelimary PR where we bump the version? Check point 7 at https://developers.home-assistant.io/docs/review-process#creating-the-perfect-pr for the explanation behind it :)

Reply:
Sorry, I opened a prelimary PR here. https://github.com/home-assistant/core/pull/130869

Reply:
Thank you for your feedback. I have made the requested changes and addressed the issues. Please review again.